Atomic/objects/container.py

```diff
@@ -104,3 +104,7 @@ def user_command(self):
     @user_command.setter
     def user_command(self, value):
         self._user_command = value
+
+    @property
+    def image(self):
+        return self.image_id
```

Comment:
Missing Return, otherwise LGTM
Do we need to get this fix into RHEL build today?